sesman/session.c

```diff
@@ -396,9 +400,8 @@ session_start_chansrv(char *username, int display)
 
         /* executing chansrv */
         g_execvp(exe_path, (char **) (chansrv_params->items));
+        
         /* should not get here */
-        LOG(LOG_LEVEL_ERROR, "error starting chansrv "
-            "- user %s - pid %d", username, g_getpid());
         list_delete(chansrv_params);
```

Comment:
Don't understand why the `LOG()` is removed. We should have a log if the `g_evecvp()` fails shouldn't we?

Reply:
I moved all of the error logging about failed `exec` calls to the `g_exec*` functions. See [os_calls.c](https://github.com/neutrinolabs/xrdp/pull/1843/commits/b530b9fa5b4d56b12d89fade9c3e2ed25d7d0e86#diff-b04475dad2718b002e94b443cb48a735bae669abe6ab10bdfc10f17f7adf1a20R2616-R2627)

Reply:
That's a much better idea.